project/src/components/favorites-screen/favorites-screen.tsx

```diff
@@ -0,0 +1,149 @@
+                        </h2>
+                        <p className='place-card__type'>Apartment</p>
+                      </div>
+                    </article>
+
+                    <article className='favorites__card place-card'>
+                      <div className='favorites__image-wrapper place-card__image-wrapper'>
+                        <a href='#!'>
+                          <img className='place-card__image' src='img/room-small.jpg' width='150' height='110' alt='Place' />
+                        </a>
+                      </div>
+                      <div className='favorites__card-info place-card__info'>
+                        <div className='place-card__price-wrapper'>
+                          <div className='place-card__price'>
+                            <b className='place-card__price-value'>&euro;80</b>
+                            <span className='place-card__price-text'>&#47;&nbsp;night</span>
+                          </div>
+                          <button className='place-card__bookmark-button place-card__bookmark-button--active button' type='button'>
+                            <svg className='place-card__bookmark-icon' width='18' height='19'>
+                              <use xlinkHref='#icon-bookmark'></use>
+                            </svg>
+                            <span className='visually-hidden'>In bookmarks</span>
+                          </button>
+                        </div>
+                        <div className='place-card__rating rating'>
+                          <div className='place-card__stars rating__stars'>
+                            <span style={{width: '80%'}}></span>
+                            <span className='visually-hidden'>Rating</span>
+                          </div>
+                        </div>
+                        <h2 className='place-card__name'>
+                          <a href='#!'>Wood and stone place</a>
+                        </h2>
+                        <p className='place-card__type'>Private room</p>
+                      </div>
+                    </article>
+                  </div>
+                </li>
+
+                <li className='favorites__locations-items'>
+                  <div className='favorites__locations locations locations--current'>
+                    <div className='locations__item'>
+                      <a className='locations__item-link' href='#!'>
+                        <span>Cologne</span>
+                      </a>
+                    </div>
+                  </div>
+                  <div className='favorites__places'>
+                    <article className='favorites__card place-card'>
+                      <div className='favorites__image-wrapper place-card__image-wrapper'>
+                        <a href='#!'>
+                          <img className='place-card__image' src='img/apartment-small-04.jpg' width='150' height='110' alt='Place' />
```

Comment:
```suggestion
                          <img 
                            className='place-card__image' 
                            src='img/apartment-small-04.jpg' 
                            width='150'
                            height='110' 
                            alt='Place' 
                          />
```

если больше двух трех пропс, лучше писать вот так, становится читабельней и понятней